### ISSUE TITLE
"rustlings" command output is unreadable when run in Windows 7 cmd

### ISSUE THREAD

**Issue by TobiasWehrum** [rust-lang/rustlings#122]
Hello,

I've just installed rustlings according to the Windows instructions. When I execute `rustlings`, here is what I see:

![cmd_2019-03-17_02-05-56](https://user-images.githubusercontent.com/1674310/54483752-392c3b00-4859-11e9-81db-6d8efd378d05.png)

I've also tried running it in bash and powershell with the same result.

**Comment by TobiasWehrum** [comment]
Alright, so what I believe to be ANSI escape sequences show up correctly in [ANSICON](http://adoxa.altervista.org/ansicon/). That's not perfect (I'd much rather use the CMD), but at least I can use it now.

**Comment by TobiasWehrum** [comment]
A few steps further down the line: Running `ansicon -i` apparently patches my system so that cmd.exe also supports the ANSI escape sequences now. I'm happy with that result.

For other people running Windows 7 it might help to either investigate how to display it properly without installing/running anything ([this StackOverflow answer](https://stackoverflow.com/a/28836703) suggests it's possible with C) or to add a note to the readme that they might want to download and run ansicon with the `-i` flag if the `rustlings` output looks all jumbled up.

**Comment by komaeda** [comment]
i would classify this as a bug if it doesn't run correctly on an unmodified windows version

**Comment by komaeda** [comment]
if anyone who runs windows could help me debug this, that'd be appreciated!

**Comment by TobiasWehrum** [comment]
Ready! What do you need?

**Comment by TobiasWehrum** [comment]
Oh, unless you need that person to actually know anything about Rust as opposed to e.g. execute and try things for you. I'm a complete newbie.

**Comment by komaeda** [comment]
@TobiasWehrum i've been wanting to remove the weird formatting anyways because it would eliminate a dependency and right now it just _looks nice and nothing else_. i'll ping you when i have a working build of that so you can try it out for me!

**Comment by barkdoll** [comment]
Windows 10 x64 => also having the same issue with the output

I will give you any information to assist with the fix just let me know what you need. 

```

       welcome to...
                 _   _ _
  _ __ _   _ ___| |_| (_)_ __   __ _ ___
 | '__| | | / __| __| | | '_ \ / _` / __|
 | |  | |_| \__ \ |_| | | | | | (_| \__ \
 |_|   \__,_|___/\__|_|_|_| |_|\__, |___/
                               |___/

?[48;2;45;45;45m?[38;2;211;208;200mThanks for installing ?[48;2;45;45;45m?[38;2;153;204;153m`?[48;2;45;45;45m?[38;2;153;204;153mrustlings?[48;2;45;45;45m?[38;2;153;204;153m`?[48;2;45;45;45m?[38;2;211;208;200m!

?[48;2;45;45;45m?[38;2;102;153;204m##?[48;2;45;45;45m?[38;2;211;208;200m ?[48;2;45;45;45m?[38;2;102;153;204mIs this your first time?

?[48;2;45;45;45m?[38;2;211;208;200mLet's make sure you're up to speed:
?[48;2;45;45;45m?[38;2;242;119;122m-?[48;2;45;45;45m?[38;2;242;119;122m ?[48;2;45;45;45m?[38;2;242;119;122mYou have Rust installed, preferably via ?[48;2;45;45;45m?[38;2;153;204;153m`?[48;2;45;45;45m?[38;2;153;204;153mrustup?[48;2;45;45;45m?[38;2;153;204;153m`
?[48;2;45;45;45m?[38;2;242;119;122m-?[48;2;45;45;45m?[38;2;242;119;122m You have ?[48;2;45;45;45m?[38;2;153;204;153m`?[48;2;45;45;45m?[38;2;153;204;153m~/.cargo/bin?[48;2;45;45;45m?[38;2;153;204;153m`?[48;2;45;45;45m?[38;2;242;119;122m added to your PATH variable
?[48;2;45;45;45m?[38;2;242;119;122m-?[48;2;45;45;45m?[38;2;242;119;122m You have cloned this repository (?[48;2;45;45;45m?[38;2;242;119;122mhttps://github.com?[48;2;45;45;45m?[38;2;242;119;122m/rust-lang/rustlings?[48;2;45;45;45m?[38;2;242;119;122m)
?[48;2;45;45;45m?[38;2;242;119;122m-?[48;2;45;45;45m?[38;2;242;119;122m You have installed Rust language support for your editor
?[48;2;45;45;45m?[38;2;242;119;122m-?[48;2;45;45;45m?[38;2;242;119;122m You have locally installed the ?[48;2;45;45;45m?[38;2;153;204;153m`?[48;2;45;45;45m?[38;2;153;204;153mrustlings?[48;2;45;45;45m?[38;2;153;204;153m`?[48;2;45;45;45m?[38;2;242;119;122m command by running:

?[48;2;45;45;45m?[38;2;211;208;200m```?[48;2;45;45;45m?[38;2;249;145;87msh
?[48;2;45;45;45m?[38;2;211;208;200mcargo install --path .
?[48;2;45;45;45m?[38;2;211;208;200m```

?[48;2;45;45;45m?[38;2;211;208;200mIf you've done all of this (or even most of it), congrats! You're ready
?[48;2;45;45;45m?[38;2;211;208;200mto start working with Rust.

?[48;2;45;45;45m?[38;2;211;208;200mTo get started, run ?[48;2;45;45;45m?[38;2;153;204;153m`?[48;2;45;45;45m?[38;2;153;204;153mrustlings verify?[48;2;45;45;45m?[38;2;153;204;153m`?[48;2;45;45;45m?[38;2;211;208;200m in order to get the first exercise.
?[48;2;45;45;45m?[38;2;211;208;200mMake sure to have your editor open!
?[0m
```

**Comment by andersarpi** [comment]
I get the same issue with rustlings 1.2.2 on Windows 10 (version 1803 for x64). Tried in both cmd and PowerShell, with the same result.

**Comment by komaeda** [comment]
This should be fixed by #167!
